ui/oopsyraidsy/data/06-ew/raid/p12s.ts

```diff
@@ -93,6 +93,22 @@ const triggerSet: OopsyTriggerSet<Data> = {
     'P12S Superchain Emission': '82DE', // partner 2x stack
     'P12S Pyre Pulse': '833A', // partner 2x stack
   },
+  triggers: [
+    {
+      id: 'P12S White Flame Share',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '82F0' }),
+      condition: (data, matches) => !data.IsImmune(matches.targetId),
+      mistake: (_data, matches) => {
+        return {
+          type: 'warn',
+          blame: matches.target,
+          reportId: matches.targetId,
+          text: matches.ability,
```

Comment:
```suggestion
          text: GetShareMistakeText(matches.ability, numTargets),
```

You'll need to `import { GetShareMistakeText } from '../../../oopsy_common';` too. This is what `AddShareTriggers` does for the `shareFail` etc helpers. Maybe I need a `GetShareMistakeFunc` that hands back that whole helper function.

One question I have for you is ... what happens if this hits two invulning tanks and a hypothetical machinist who stands too close? Should it just say `(share x3)` on the machinist (but nobody else)? Or something else?